src/internal_events/aws_cloudwatch_logs_subscription_parser.rs

```diff
@@ -12,6 +12,7 @@ impl InternalEvent for AwsCloudwatchLogsSubscriptionParserError {
         error!(
             message = "Event failed to parse as a CloudWatch Logs subscription JSON message.",
             error = ?self.error,
+            error_code = "parsing_cloudwatch_logs",
```

Comment:
```suggestion
            error_code = "parsing_failed",
```

I might generalize these a bit to share the error codes across components where it makes sense.

Reply:
I don't really agree with this because it would be the same as `error_type`.
If we take the use case of searching errors in the datadog interface for example, filtering by stage would be a first filtering (global), the error type should filter event more (in what step of the component processing it happens) and, to me, the error code should be something specific to that part of code and therefore, we shouldn't really share it between components 🤔